src/pages/error/error-page.tsx

```diff
@@ -1,14 +1,17 @@
 import { ThemeProvider } from '@mui/material/styles';
 import { Container } from "@mui/material";
 import  darkTheme  from "../../shared/themes/dark-theme"
+import { Menu } from "../../shared/components/menu"
+import { CssBaseline } from '@mui/material';
+const centering = {display: 'flex', flexDirection: 'column', alignItems: "center"}
```

Comment:
nie ma potrzeby przenosić tego do zmiennej, tym bardziej nie trzymaj jej tak blisko importów. Jak już byś tak chciał to ze 2 entery pasowałoby wjjebac